misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -57,7 +57,7 @@ internal class RealTransacter private constructor(
 
   override fun isCheckEnabled(check : Check): Boolean {
     val session = threadLocalSession.get()
-    return session != null && !session.disabledChecks.contains(check)
+    return session == null || !session.disabledChecks.contains(check)
```

Comment:
This will enable all checks for raw Hibernate sessions, which was the behaviour previously. If we have a need to disable these checks for Hibernate sessions, we can tackle that in the future. But it's easier to disable a check on something that expects it, than to enable a check on something that doesn't.

Reply:
I agree, I think it makes sense.